train.py

```diff
@@ -364,7 +404,8 @@ def trainCNN(options,args):
     if not os.path.exists(model_name):
       os.mkdir(model_name)
     os.chdir(model_name)
-   
+    shutil.copyfile("../../tests/verify_emd.py", "verify_emd.py")
```

Comment:
@therwig Is this needed?

Reply:
its not needed anymore. was a test/hack. I'll remove it